src/mixed-reality/azext_mixed_reality/custom.py

```diff
@@ -1,32 +1,182 @@
-# --------------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+import json
+from ._client_factory import cf_spatial_anchor_account, cf_remote_rendering_account
 
 
-def create_spatial_anchors_account(
-        client,
-        resource_group_name, spatial_anchors_account_name, location=None, tags=None,
-        custom_headers=None, raw=False, **operation_config):
-    return client.create(
-        resource_group_name, spatial_anchors_account_name, location, tags,
-        custom_headers, raw, **operation_config)
+def spatial_anchor_account_list(cmd,
+                                resource_group_name=None):
+    client = cf_spatial_anchor_account(cmd.cli_ctx)
```

Comment:
Declare `command_type` and `client_factory` in `commands.py`, there is no need to obtain the client separately

Reply:
I think both of ways are ok